src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ReleaseViewModels.cs

```diff
@@ -51,18 +52,31 @@ public class ReleaseViewModel
         [JsonConverter(typeof(StringEnumConverter))]
         public ReleaseType Type { get; set; }
 
-        public Contact Contact { get; set; }
+        public Contact? Contact { get; set; }
```

Comment:
Same as above - why has this been made optional?

Reply:
See previous comment.